planning.md

```diff
@@ -5,10 +5,21 @@ sidebar_link: false
 
 We are always interested in ideas for new events. It could be a presentation night on a particular topic, or a new kind of event altogether.  Many of our current events were first proposed by members.
 
-To start planning an event:
+### To start planning an event:
 
-1. Join us in the **#organizing** channel in [our Slack workspace](slack.md).  Start a conversation along the lines of "I'm thinking about XYZ," and get a discussion going about what shape the event could take.
+1. Fill out the [event proposal form](https://forms.gle/ymyPXJkn8MbThadZA) here with some details of what you're thinking about, or make a post in the **#organizing** channel in [our Slack workspace](slack.md) with the same information. This does not have to be a complete plan! Just the seed of an idea is great - we are happy to help you expand on it.
```

Comment:
I'm not sure we all wanted to go with the form.  One the one hand it could seem like paperwork, I guess on the other, it's intimidating to walk cold into a Slack channel where others are watching.